### DISCUSSION TITLE
only 2 macOS vm can be created in apple Silicon? Is that true?

### DISCUSSION THREAD

**Discussion by jk2K** [cirruslabs/tart#507] [Q&A] [answered]
I found the article https://eclecticlight.co/2022/08/04/virtualisation-on-apple-silicon-macs-8-how-apple-limits-vms/

one machine can only run 2 macOS vm, for iOS CI and DevOps, unable to create more macOS vm by purchasing Ultra chip(20 core cpu)

https://www.apple.com/mac-studio/compare/?modelList=Mac-studio-2022,Mac-mini-M2,MacBook-Pro-16-M2

**Comment by edigaryev** [comment, marked as answer]
This is true and in accordance with [macOS Ventura EULA](https://www.apple.com/legal/sla/docs/macOSVentura.pdf):

>(iii) to install, use and run up to two (2) additional copies or instances of the Apple Software, or any prior macOS or OS X operating system software or subsequent release of the Apple Software, within virtual operating system environments on each Apple-branded computer you own or control that is already running the Apple Software, for purposes of: (a) software development; (b) testing during software development; (c) using macOS Server; or (d) personal, non-commercial use.

Due to this policy it makes very little sense in buying more powerful Mac machines for virtualization at the moment, unless your workloads actually need that much amount of resources.

**Comment by syscod3** [comment]
does this apply to OSX only? for instance, can I have more linux VMs?

**Reply by fkorotkov** [reply]
It seems you can. I was able to run 3 Ubuntu VMs concurrently.
